doc/database-model.md

```diff
@@ -0,0 +1,61 @@
+
+# Database Entity Relationship Diagram
+
+```mermaid
+erDiagram
+    USER {
```

Comment:
what happend with dates? i want to know when user starts to use the app

Reply:
timestamps are automatically created if set true